posts/2024-08-27-quarto-on-HPC/index.qmd

```diff
@@ -0,0 +1,167 @@
+---
+title: "Quarto on HPC"
+image: https://quarto.org/quarto.png
+lightbox: true
+description: "A tutorial for setting up Quarto in a remote server (HPC)"
+author:
+  - name: Fernando Cervantes
+    url: https://www.github.com/fercer
+    orcid: 0000-0003-0652-2750
+    affiliation: The Jackson Laboratory
+    affiliation-url: https://jax.org/ 
+date: 2024-08-27
+categories: [HPC,Tutorials] # self-defined categories
+draft: false # setting this to `true` will prevent your post from appearing on your listing page until you're ready!
+---
+
+# What is Quarto?
+
+From the [Quarto](https://quarto.org) webpage, it is
+
+> An open-source scientific and technical publishing system
+
+Quarto can be used to create documentation pages, scientific notebooks, professional presentations, and websites (like this one!).
+
+This system is useful for creating notes, particularly these containing Python/R `code`.
+And in contrast with jupyter notebooks, it offers a wide variety of ways to share these notes.
+
+# Install Quarto on a remote server (HPC)
+
+In this post, two options are presented to install Quarto on a HPC server.
+The first option installs Quarto in your home directory and works fine when creating code-less documents.
+
+The second option is a way to have Quarto installed in a virtual environment, which is useful for creating jupyter-like notebooks with executable code.
+
+## Option 1. Install Quarto on your home directory
+
+Download the Quarto source code for *Linux x86 Tarball* from their [official website](https://quarto.org/docs/download/tarball.html?version=1.5.56&idPrefix=download) into your home directory.

```

Comment:
Oh one more thought: this link has the version number in it, so it could become stale with a new version. Maybe just link to the page itself?
```suggestion
Download the Quarto source code for *Linux x86 Tarball* from their [official website](https://quarto.org/docs/download/tarball.html) into your home directory.

```
https://quarto.org/docs/download/tarball.html

Reply:
I didn't notice that the link contained the specific version when I copied it since it links to the general tarball download page.
Changing the link to the generic tarball download webpage a suggested!